crmsh/ui_sbd.py

```diff
@@ -89,8 +89,19 @@ class SBD(command.UI):
     - sbd purge
     '''
     name = "sbd"
-    TIMEOUT_TYPES = ("watchdog", "allocate", "loop", "msgwait", "crashdump-watchdog")
-    DISKLESS_TIMEOUT_TYPES = ("watchdog", "crashdump-watchdog")
+    TIMEOUT_TYPES = {
```

Comment:
It would be better if the name can describe what the values is. For example, `TIMEOUT_TYPE_MINIMUMS`